src/main/resources/json/coordinates.json

```diff
@@ -0,0 +1,4 @@
+{
+  "latitude": 0.0,
```

Comment:
How dows this Json help us?

Reply:
Zum Testen des LocoJsonMappers. Der transportiert bei OSM auch Koordinaten als Json.